lizard_auth_client/views.py

```diff
@@ -467,11 +467,11 @@ def dispatch(self, request, *args, **kwargs):
 
     @property
     def active_users(self):
-        return User.objects.filter(is_active=True)
+        return User.objects.filter(is_active=True).order_by('email')
```

Comment:
Ik heb even in django gekeken: er zit alleen een index op de username (indirect, omdat daar unique=True bij staat).

Maar goed, zo super veel gebruikers zullen er niet zijn dat dit een probleem wordt. En anders mag de site zelf iets beters verzinnen :)